src/main/java/org/osiam/resources/exceptions/OsiamBackendFailureException.java

```diff
@@ -0,0 +1,8 @@
+package org.osiam.resources.exceptions;
+
+public class OsiamBackendFailureException extends OsiamException {
+
```

Comment:
No there doesn't need to be a `serialVersionUUID`. The only OSIAM exception in need of a `serialVersionUUID` is the `InvalidConstraintException` because that's the only one that has a structure different  from the exception it inherits from. So I removed them all except there.